cmd/ttn-lw-cli/commands/root.go

```diff
@@ -133,6 +134,14 @@ func preRun(tasks ...func() error) func(cmd *cobra.Command, args []string) error
 			}
 		}
 
+		// Drop explicit `:443` port number from OAuth server address if present.
+		// Causes issues with `--http.redirect-to-tls` stack option.
+		u, err := url.Parse(config.OAuthServerAddress)
+		if u.Port() == "443" {
+			u.Host = u.Hostname()
+			config.OAuthServerAddress = u.String()
+		}
```

Comment:
I think it's better to also check the scheme. If it's `http` on `80` or `https` on `443` we can remove the port.